src/Controller/CandidatController.php

```diff
@@ -17,19 +17,18 @@ class CandidatController extends AbstractController
 {
     /**
      * @Route("/profil/{id<^[0-9]+$>}", name="show")
```

Comment:
maintenant que tu as relier user et candidat, tu peux te permettre d'enlever l'id dans la route et le param converter avec $candidat, et récupérer directement $this->getUser(), ou $this->getUser()->getCandidate() je ne sais pas ce qui est le plus pratique
pareil, pas besoin de recup hobbies car tu devrais pouvoir faire un user.candidate.hobbies dans twig directement ;-)

Reply:
La table hobbies n'était pas encore reliée a candidat car je n'avais ni user ni candidat quand j'ai fait la user story. DU coup l'affichage d'hobbies arrive dans un fix à part de celui là